packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -2597,6 +2597,84 @@ module.exports = function (config, DB) {
       })
     })
 
+    describe('account subscriptions', () => {
+      const subs = [...Array(10)].map(() => 'sub-' + Math.random())
```

Comment:
Can you rename `subs` to `subscriptions`, [`sub` has a specific meaning in JWTs (Subject claim)](https://tools.ietf.org/html/rfc7519#section-4.1.2) and since we deal with both, I can imagine getting confused. Yeah, I know context and all, but...

Reply:
Renaming this to `subscriptionIds` to be even more specific